common-service-core/src/main/java/com/ca/mfaas/message/api/ApiMessageView.java

```diff
@@ -16,19 +16,13 @@
 import java.util.List;
 
 /**
- * This class is intended for REST API responses that contain error,
- * warning, or informational messages in the common Apiml format.
+ * The class is intended for REST API responses that contain error,
+ * warning, or informational messages in the common MFaaS format.
```

Comment:
Why Mfaas?

Reply:
It is just result of the merge. I have changed to Apiml